packages/client/components/ActivityLibrary/ActivityLibrary.tsx

```diff
@@ -62,9 +105,34 @@ export const ActivityLibrary = (props: Props) => {
   const {
     query: searchQuery,
     filteredItems: filteredTemplates,
-    onQueryChange
+    onQueryChange,
+    resetQuery
   } = useSearchFilter(templates, getTemplateValue)
 
+  const history = useHistory()
+  const {match} = useRouter<{categoryId?: string}>()
+  const {params} = match
+  const {categoryId: selectedCategory} = params
+
+  useEffect(() => {
+    if (!selectedCategory || !Object.keys(CATEGORY_ID_TO_NAME).includes(selectedCategory)) {
+      history.replace(`/activity-library/category/${QUICK_START_CATEGORY_ID}`)
+    }
+  }, [selectedCategory, history])
```

Comment:
+1 one thing, as I'm not a fan of `useEffect` I'd use `<Redirect />` here 😛

Reply:
Nice call! Updated